File: 02.CLIWEB/CLIWEB_COMER_GR08/src/main/java/ec/edu/monster/model/Credito.java
package ec.edu.monster.model;

import java.math.BigDecimal;

public class Credito {
    private Integer idCredito;
    private String cedula;
    private BigDecimal montoSolicitado;
    private BigDecimal montoAprobado;
    private Integer plazoMeses;
    private BigDecimal tasaAnual;
    private BigDecimal cuotaFija;
    private String estado;
    private String fechaSolicitud;
    private String fechaAprobacion;

    public Integer getIdCredito() {
        return idCredito;
    }

    public void setIdCredito(Integer idCredito) {
        this.idCredito = idCredito;
    }

    public String getCedula() {
        return cedula;
    }

    public void setCedula(String cedula) {
        this.cedula = cedula;
    }

    public BigDecimal getMontoSolicitado() {
        return montoSolicitado;
    }

    public void setMontoSolicitado(BigDecimal montoSolicitado) {
        this.montoSolicitado = montoSolicitado;
    }

    public BigDecimal getMontoAprobado() {
        return montoAprobado;
    }

    public void setMontoAprobado(BigDecimal montoAprobado) {
        this.montoAprobado = montoAprobado;
    }

    public Integer getPlazoMeses() {
        return plazoMeses;
    }

    public void setPlazoMeses(Integer plazoMeses) {
        this.plazoMeses = plazoMeses;
    }

    public BigDecimal getTasaAnual() {
        return tasaAnual;
    }

    public void setTasaAnual(BigDecimal tasaAnual) {
        this.tasaAnual = tasaAnual;
    }

    public BigDecimal getCuotaFija() {
        return cuotaFija;
    }

    public void setCuotaFija(BigDecimal cuotaFija) {
        this.cuotaFija = cuotaFija;
    }

    public String getEstado() {
        return estado;
    }

    public void setEstado(String estado) {
        this.estado = estado;
    }

    public String getFechaSolicitud() {
        return fechaSolicitud;
    }

    public void setFechaSolicitud(String fechaSolicitud) {
        this.fechaSolicitud = fechaSolicitud;
    }

    public String getFechaAprobacion() {
        return fechaAprobacion;
    }

    public void setFechaAprobacion(String fechaAprobacion) {
        this.fechaAprobacion = fechaAprobacion;
    }
}

File: SERVIDOR SOAP/ws_banquito_dotnet_gr08/ec.edu.monster.model/Credito.cs
using System;

namespace ec.edu.monster.model
{
    public class Credito
    {
        public int IdCredito { get; set; }
        public string Cedula { get; set; }
        public decimal MontoSolicitado { get; set; }
        public decimal MontoAprobado { get; set; }
        public int PlazoMeses { get; set; }
        public decimal TasaAnual { get; set; }
        public decimal CuotaFija { get; set; }
        public string Estado { get; set; }
        public DateTime FechaSolicitud { get; set; }
        public DateTime? FechaAprobacion { get; set; }

        public bool EstaActivo()
        {
            return string.Equals(Estado, "ACTIVO", StringComparison.OrdinalIgnoreCase);
        }

        public void Activar()
        {
            Estado = "ACTIVO";
        }
    }
}
